runtime/oti/VMHelpers.hpp

```diff
@@ -2052,16 +2052,22 @@ class VM_VMHelpers
 	}
 
 #if JAVA_SPEC_VERSION >= 19
+	static VMINLINE ContinuationState *
```

Comment:
I'd prefer we return 'volatile ContinuationState *'  and consequently change anywhere in the code where we really store this address into a local to be also the same volatile type, and most of other helpers to pass volatile pointer too (compiler will impose it what's needed and what's not)

Reply:
agree, I think in GC code historically we use more the former, but I believe VM tends to use more later, so yes, let's go with latter